app/posts/claim-general-mentor-training-funding/2022-06-26-general-mentor-training-grant-guidance-and-claim-form.md

```diff
@@ -0,0 +1,123 @@
+---
+title: General mentor training grant guidance and claim form
+description: Guidance page summarising ITT gerneral mentor funding and claim form.
+date: 2022-06-26
+---
+
+
+[The initial teacher training (ITT) market review report](https://www.gov.uk/government/publications/initial-teacher-training-itt-market-review-report) was published in July&nbsp;2021. After public consultation, the DfE published their [response to the ITT market review](https://www.gov.uk/government/consultations/initial-teacher-training-itt-market-review) in December&nbsp;2021.
+
+The recommendations which need to be followed from the 2024/25 academic year onwards, include requirements for:
+
+- mentors to train take part in 20 hours of initial training and 6 hours of top-up training in subsequent years
+- ITT providers to have lead mentors or a lead mentorship team
+- trainees to take part in ’Intensive training and practice’ sessions (4 weeks for postgraduate courses and 6 weeks for undergraduate courses)
+
+## ITT reform funding
+
+To support providers and schools in meeting the new requirements the DfE is making grant funding available.
+
+For schools the funding is to help cover the cost of classroom cover while mentors train. In the 2024/25 academic year, general mentors will need to complete 20&nbsp;hours of initial mentor training. In subsequent years general mentors who have completed the 20&nbsp;hours will need 6&nbsp;hours of training.
+
+[See full details of the ITT reform funding.](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1077177/Initial_Teacher_Training__ITT__Reform_Funding_Guidance_160522.pdf)
+
+## Prototype
+
+One of the streams of work for the mentoring team is looking into how the grants will be distributed.
+
+We plan to distribute the funding for ITT providers through DfE’s normal business processes.
+
+Because the general mentor training grant is available to more than 20,000&nbsp;schools and we need specific information for each claim, a digital form is necessary to collect the data.
+
+To test our questions and assumptions about how the service will work, we built a guidance page and a form (in Google forms).
+
+### Guidance
+
+![GOV.UK page with summary of ITT general mentor training grant](general-mentor-training-grant-guidance.png "Apply for ITT general mentor training funding")
+
+### Claim form
+
+![Google form asking user for their email address](google-form/01-email-address.png "Claim form — email address")
```

Comment:
Prefacing these captions with `Claim form –` feels redundant. Perhaps make the captions in to sentences?